ktlint-cli-reporter-baseline/build.gradle.kts

```diff
@@ -9,3 +9,26 @@ dependencies {
 
     testImplementation(projects.ktlintTest)
 }
+
+publishing {
+    publications {
+        // TODO: Remove in release after ktlint 1.0.1
+        create<MavenPublication>("relocation-ktlint-cli-reporter-baseline") {
```

Comment:
Not be familiar with maven relocation, but if you wan't to sign this, could add something like:

```kotliln
signing {
    sign(publishing.publications["relocation-ktlint-cli-reporter-baseline"])
}
```

Ref: https://github.com/pinterest/ktlint/blob/ea8df7e85abb85d14deff59dd056357bb5700feb/build-logic/src/main/kotlin/ktlint-publication.gradle.kts#L90

Reply:
My question is not specifically about the maven relocation, but how to avoid code duplication in Gradle build. I have now added a `publication` and `signing` block in each module for which the maven coordinate has been changed. It is good enough as it will be removed soon.